homeassistant/components/venstar/const.py

```diff
@@ -5,6 +5,7 @@
 
 ATTR_FAN_STATE = "fan_state"
 ATTR_HVAC_STATE = "hvac_mode"
+ATTR_SCHED_PART = "schedule_part"
```

Comment:
why is there a mismatch between there being an underscore here and no underscore in `climate.py`?

Reply:
The Venstar API and, as a result, the venstarcolortouch library that Home Assistant is using, don't separate words in attribute names (similarly you can see `self._client.fanstate` in climate.py, where `_client` is an instance of a type from the venstarcolortouch library). However, the other attribute names here do separate words with underscores, so I followed that pattern.

I suspect that this is because the attribute names are user-facing in Home Assistant and this is the style they've adopted, whereas the underlying library is just copying the Venstar API's style, which is clearly not meant for the eyes of users.